src/makeRobotAccountant.js

```diff
@@ -6,7 +6,20 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+  const phrase = 'Bzzz... Error!';
+
+  return (a) => {
+    return (b) => {
+      count++;
+
+      return (
+        count > 3 && (count % 2 === 0)
+          ? phrase
+          : Number(a + b)
```

Comment:
`a` and `b` always number here, so no need to transform them to `number` by hand
anyway if it would be the string you'd add (concat) them as strings and just then transform the result to a number:
![image](https://user-images.githubusercontent.com/86923089/204038032-934715f1-4db1-4cc4-83f5-df2691ed28cb.png)